src/page/product-list.page.ts

```diff
@@ -0,0 +1,24 @@
+import { $, ElementFinder } from 'protractor';
+import { ProductAddedModalPage } from './product-added-modal.page';
+
+export class ProductListPage{
+  private inStockbutton: ElementFinder;
+  private addToCartButton: ElementFinder;
+
+  constructor () {
+    this.inStockbutton = $('#center_column > ul > li > div > div.right-block > span > span');
+    // tslint:disable-next-line:max-line-length
+    this.addToCartButton = $('#center_column > ul > li > div > div.right-block > div.button-container >' +
+    'a.button.ajax_add_to_cart_button.btn.btn-default > span');
```

Comment:
i think we can search for another selector, like 
.ajax_add_to_cart_button

Reply:
Locator Updated